lib/entities/entry.ts

```diff
@@ -71,3 +72,11 @@ export function wrapEntry(makeRequest: MakeRequest, data: EntryProps): Entry {
  * @private
  */
 export const wrapEntryCollection = wrapCollection(wrapEntry)
+
+/**
+ * @private
+ */
+export const wrapEntryTypeCursorPaginatedCollection: (
+  makeRequest: MakeRequest,
+  data: CursorPaginatedCollectionProp<EntryProps>,
+) => CursorPaginatedCollectionProp<EntryProps> = wrapCursorPaginatedCollection(wrapEntry)
```

Comment:
Type should be infered.

```suggestion
export const wrapEntryTypeCursorPaginatedCollection  = wrapCursorPaginatedCollection(wrapEntry)
```